### PR TITLE
(nobug) Make button for collapsing devtools less prominent

### PR DESCRIPTION
**Pull request by k88hudson**
At aaron's request, slight design change for the devtools button.

image

### CODE REVIEW

**Comment by k88hudson** [comment]
Oops just saw e35dec51aa0162cfe82ab5bdbf0f6a24d7f53034, going to ask if aaron still wants this

**Comment by k88hudson** [comment]
He does still want the change

File: content-src/components/ASRouterAdmin/ASRouterAdmin.jsx [hunk 1]
```diff
@@ -805,11 +805,14 @@ export class CollapseToggle extends React.PureComponent {
...
     const {props} = this;
     const {renderAdmin} = this;
-    const action = (this.state.collapsed || !renderAdmin) ? "Expand" : "Collapse";
+    const label = `${(this.state.collapsed || !renderAdmin) ? "Expand" : "Collapse"} devtools`;
     return (<React.Fragment>
       <a href="#devtools"
-        className={`asrouter-toggle asrouter-${action.toLowerCase()}`}
-        onClick={this.renderAdmin ? this.onCollapseToggle : null}>{action} Devtools</a>
+        title={label}
+        className={`asrouter-toggle ${this.state.collapsed ? "collapsed" : "expanded"}`}
```
Comment (Mardak) [inline content-src/components/ASRouterAdmin/ASRouterAdmin.jsx]: Looks like state.collapsed is default false even on a plain about:newtab so it renders the className as expanded (resulting in background). Probably just needs the `|| !renderAdmin` as above.

Comment (k88hudson) [reply content-src/components/ASRouterAdmin/ASRouterAdmin.jsx]: ah, good point

Comment (rlr) [inline content-src/components/ASRouterAdmin/ASRouterAdmin.jsx]: I think you need `this.state.collapsed || !renderAdmin` here like above for the label. Right now it's showing the expanded background on the first render.

Comment (rlr) [reply content-src/components/ASRouterAdmin/ASRouterAdmin.jsx]: Or maybe that's intentional?

Comment (rlr) [reply content-src/components/ASRouterAdmin/ASRouterAdmin.jsx]: gah. Ed beat me.

**Pull request merged**